pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java

```diff
@@ -3875,6 +3875,13 @@ public void updateArray(String columnName, @Nullable Array x) throws SQLExceptio
         throw new PSQLException(GT.tr("conversion to {0} from {1} not supported", type, getPGType(columnIndex)),
                 PSQLState.INVALID_PARAMETER_VALUE);
       }
+    } else if (type == byte[].class) {
+      if (sqlType == Types.BINARY) {
```

Comment:
How about `Types.VARBINARY`, `Types.LONGVARBINARY`?

Reply:
I thought about it, but decided against because Blob only checks for BINARY.  Besides, here the type is what PG thinks is appropriate for BYTEA, its not a user input, so actually it should never (?) be multiple values.Do you think

Reply:
If someone stored `VARBINARY` or `LONGVARBINARY` the driver still needs to be able to handle it